### ISSUE TITLE
Unable to update pins for R 3.4.1 on win-64

### ISSUE THREAD

**Issue by jdblischak** [conda-forge/r-base-feedstock#40]
**tl;dr We haven't been able to update the pins for R 3.4.1 on Windows.**

I was recently notified by @cbrueffer that R packages were failing for R 3.4.1: https://github.com/conda-forge/staged-recipes/pull/5405#issuecomment-373317764. I was surprised because I thought nothing had changed for R 3.4.1, but then I remembered today that PR #39 from @jakirkham was merged by @bgruening a few days ago. It was just a straightforward rerendering, but surprisingly this caused new binaries for win-64 and win-32 to be uploaded to Anaconda Cloud:

https://anaconda.org/conda-forge/r-base/files?version=3.4.1

These packages hadn't previously been uploaded because the merge commit from @ocefpaf in PR #27 did not trigger a Windows build (I have no idea why):

<img width="1012" alt="screen shot 2018-03-19 at 1 55 22 pm" src="https://user-images.githubusercontent.com/1608317/37612987-3209828c-2b7d-11e8-9e24-b2a63e256c0d.png">

Now that a Windows build was triggered, the build number 3 binaries appear to be failing for the same reason that the build number 2 binaries were previously failing (and were subsequently labeled as broken): #26

**Action items:**

- [x] Mark win-64/r-base-3.4.1-3.tar.bz2 and win-32/r-base-3.4.1-3.tar.bz2 as broken
- [x] Configure pins so that R 3.4.1 can build packages on win-64
- [ ] Update r-base recipe to skip win-32 (we skip win-32 for all R package recipes, so it seems unnecessary to build r-base for win-32)

**Comment by jdblischak** [comment]
I realize that my initial message did not make it clear that this is **currently breaking all conda-forge R packages** from building Windows packages since 5 days ago (2018-03-14), e.g.

* [PRs to staged-recipes](https://github.com/conda-forge/staged-recipes/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+created%3A%3E%3D2018-03-14+r-+)
* [PRs from regro-cf-autotick-bot](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+author%3Aregro-cf-autotick-bot+archived%3Afalse+created%3A%3E%3D2018-03-14+r-+)

**Comment by jakirkham** [comment]
For a non-staged-recipes example, here's [one build for `r-rcpp`]( https://ci.appveyor.com/project/conda-forge/r-rcpp-feedstock/build/1.0.9/job/ngbb5ttutw27rm6k ).

**Comment by jakirkham** [comment]
Thanks for letting us know about this problem.

Have marked the two latest builds R 3.4.1 as broken.

Would be good to understand why they are broken. Pinning changes alone causing this issue are a bit surprising.

**Comment by jdblischak** [comment]
Thanks @jakirkham!

> Would be good to understand why they are broken. Pinning changes alone causing this issue are a bit surprising.

Agreed, but the only thing that has changed between build 1 and the subsequent builds 2 and 3 are the pins. Build 1 can build R packages on win-64, but builds 2 and 3 can't (they get the [timezone error](https://github.com/conda-forge/staged-recipes/pull/5405#issuecomment-373718358)).

**Comment by jdblischak** [comment]
@cbrueffer Could you please try restarting some of your AppVeyor builds to confirm this short-term solution is working?

**Comment by jakirkham** [comment]
Sounds like based on your other comment that a newer version of R might fix this, correct? If so, maybe it is worth an experiment?

**Comment by jdblischak** [comment]
> Sounds like based on your other comment that a newer version of R might fix this, correct? If so, maybe it is worth an experiment?

@jakirkham What did you have in mind? We would need to not only build r-base with a newer version, but also test that it is able to build R packages on win-64.

**Comment by cbrueffer** [comment]
Thanks for looking into this! https://github.com/conda-forge/staged-recipes/pull/5402 successfully built, so it appears the problem is fixed for now. So the timezone stuff is the likely culprit?

**Comment by jdblischak** [comment]
@ocefpaf Do you have any thoughts on how to update the pins such that they will work on win-64?

**Comment by ocefpaf** [comment]
Sorry @jdblischak I do not. I'll try to look at this over the weekend but I cannot promise. (Day job is quite demanding these day.)

**Comment by jdblischak** [comment]
@ocefpaf No worries! I appreciate you're busy. I wanted to check if you had any ideas since you've been the main person bumping  the pinned dependencies for r-base.

**Comment by isuruf** [comment]
Finally figured out what was wrong. URL at https://github.com/conda-forge/r-base-feedstock/blob/master/recipe/build.sh#L244 is no longer working and the installation process fails without an error code when miktex can't be found.

**Comment by jdblischak** [comment]
@isuruf Great! Should we try re-building it with an updated download path? My guess would be:

http://ctan.mirrors.hoobly.com/systems/win32/miktex/setup/windows-x86/miktex-portable-2.9.6621.exe

**Comment by isuruf** [comment]
@jdblischak, my preference is to update miktex-feedstock and use that in the r-base.

**Comment by isuruf** [comment]
@jdblischak, I've tried using the miktex conda package, but it didn't work. Let's just change the url in build.sh

**Comment by jdblischak** [comment]
> Let's just change the url in build.sh

@isuruf I've attempted this in PR #44

**Comment by jdblischak** [comment]
Fixed by #44. Skipping the win-32 build can be done later since it's not hurting anything.

A big thanks to @isuruf for solving the problem!
